### ISSUE TITLE
Missing translation for error message

### ISSUE THREAD

**Issue by aslmx** [fossar/selfoss#1387]
There is a missing translation for the following text

`Es können nicht alle sichtbaren Elemente markiert werden: **Offline storage is not available**`

Suggestion:

`Es können nicht alle sichtbaren Elemente markiert werden: Offline Speicher ist nicht verfügbar`

**Comment by jtojnar** [comment]
Yeah, the error message comes from an exception that should not really be displayed. But the client side model is currently so tangled it probably is not easily fixable.
